src/visitor.rs

```diff
@@ -439,6 +438,7 @@ impl Visitor {
                             self.visit_generic_param_defs(path, generic_params)?;
                         }
                         GenericBound::Outlives(_) => {}
+                        GenericBound::Use(_) => {},
```

Comment:
What is this? Does it refer to any types we need to check?

Reply:
It'll be related to the new precise capture syntax. See https://rust-lang.github.io/rfcs/3617-precise-capturing.html?highlight=use